adoc/chapters/programming_interface.adoc

```diff
@@ -7671,6 +7694,12 @@ The [code]#host_accessor# class supports the following access modes:
 [code]#access_mode::read#, [code]#access_mode::write# and
 [code]#access_mode::read_write#.
 
+A [code]#host_accessor# to a buffer is only a view over that buffer.
+Therefore, it is the application's responsibility to ensure that the lifetime
+of the buffer is at least as long as the lifetime of the host accessor.
+Calling any member function on a host accessor after its associated buffer has
+been destroyed results in undefined behavior.
```

Comment:
I do not understand how it is an issue in a conforming implementation.
If you you have a live accessor, the buffer is still kept alive though for example an internal `shared_ptr`.

Reply:
The case of the host accessor looks simpler, very synchronous without any lazy lambda evaluation and should keep a buffer internal state alive.